DataConnectors/NXLogLinuxAudit.json

```diff
@@ -0,0 +1,94 @@
+{
+    "id": "NXLogLinuxAudit",
+    "title": "NXLog LinuxAudit",
+    "publisher": "NXLog",
+    "descriptionMarkdown": "The NXLog [LinuxAudit](https://nxlog.co/documentation/nxlog-user-guide/im_linuxaudit.html) data connector supports custom audit rules and collects logs without auditd or any other user-space software. IP addresses and group/user ids are resolved to their respective names making [Linux audit](https://nxlog.co/documentation/nxlog-user-guide/linux-audit.html) logs more intelligible to security analysts. This REST API connector can efficiently export Linux security events to Azure Sentinel in real-time.",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "LinuxAudit_CL",
+            "baseQuery": "LinuxAudit_CL"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Most frequent type",
+            "query": "LinuxAudit_CL\n| summarize EventCount = count() by type_s \n| where strlen(type_s) > 1 \n| render barchart"
+        },
+        {
+            "description" : "Most frequent comm",
+            "query": "LinuxAudit_CL\n| summarize EventCount = count() by comm_s\n| where strlen(comm_s) > 1\n| render barchart"

```

Comment:
Could not find field with name comm_s. Can you please fix this query?

Reply:
@v-jayakal This was resolved in the latest commit [08dec290](https://github.com/Azure/Azure-Sentinel/commit/08dec290af28748496d5ed66779cf2fef6718dab).